source/docs/zero-to-robot/step-2/frc-game-tools.rst

```diff
@@ -4,193 +4,164 @@ Installing the FRC Game Tools
 =============================
 
 The FRC Game Tools contains the following software components:
-LabVIEW Update, FRC Driver Station, and FRC Utilities. The
-LabVIEW runtime components required for the Driver Station and
-Utilities are included in this package. **No components from the
-LabVIEW Software for FRC package are required for running either the Driver
-Station or Utilities.**
 
-.. note:: The Driver Station will only work on Windows 7, Windows 8,\
- Windows 8.1, and Windows 10. It will not work on Windows XP.
+- LabVIEW Update
+- FRC Driver Station
+- FRC Utilities.
+
+The LabVIEW runtime components required for the Driver Station and Utilities are included in this package.
+
+.. note:: No components from the LabVIEW Software for FRC package are required for running either the Driver Station or Utilities.
+
+Requirements
+------------
+
+- Windows 7 or higher (Windows 7, 8, 8.1, 10). Windows 10 is the recommended OS.
+- Download the update from `NI <https://www.ni.com/en-us/support/downloads/drivers/download.frc-game-tools.html>`__.
+
+.. image:: images/labview/offline-installer.png
+
+If you wish to install on other machines offline, do not click the :guilabel:`Download` button, click :guilabel:`Individual Offline Installers` and then click :guilabel:`Download`, to download the full installer.
 
 Uninstall Old Versions (Recommended)
 ------------------------------------
 
-.. Warning:: LabVIEW teams have already completed this step, do not repeat it.
-
-.. Note:: It is only necessary to uninstall previous versions when installing a new year's tools. For example, uninstall the 2019 tools before installing the 2020 tools. It is not necessary to uninstall before upgrading to a new update of the 2020 game tools.
+.. warning:: LabVIEW teams have already completed this step, do not repeat it.
```

Comment:
This seems a bit light to be a warning. Doesn't have to be `note`, but should be lighter than `warning`.
```suggestion
.. note:: LabVIEW teams have already completed this step, do not repeat it.
```

Reply:
I disagree. Reinstalling LabVIEW because you accidentally uninstalled it is a pretty big deal. I think this deserves a warning

Reply:
Alternative suggestion, use ``.. important::``. I think it's better for this anyways.